subsys/shell/modules/kernel_service.c

```diff
@@ -44,13 +44,45 @@ static int cmd_kernel_version(const struct shell *sh,
 	return 0;
 }
 
+#define MINUTES_FACTOR (MSEC_PER_SEC * 60)
+#define HOURS_FACTOR   (MINUTES_FACTOR * 60)
+#define DAYS_FACTOR    (HOURS_FACTOR * 24)
+
 static int cmd_kernel_uptime(const struct shell *sh,
 			     size_t argc, char **argv)
 {
 	ARG_UNUSED(argc);
 	ARG_UNUSED(argv);
 
-	shell_print(sh, "Uptime: %u ms", k_uptime_get_32());
+	uint32_t milliseconds = k_uptime_get_32();
```

Comment:
what about using `k_uptime_get_64` as 32 bit limits to 49 days.

Reply:
@nordic-krch  done